docs/src/adoc/index.adoc

```diff
@@ -2603,6 +2603,47 @@ and setter parameters.
 - `@BindJpa` and `JpaMapper` respect qualifiers on getters and setters.
 - `@BindKotlin`, `bindKotlin()`, and `KotlinMapper` respect qualifiers on constructor parameters, getters, setters, and setter parameters.
 
+==== Custom qualifiers
+
+There are always 2 parties involved in using qualifiers:
+the target (method, parameter, etc) and the handler (`ArgumentFactory`, etc). Generally, both need to be _qualified_ identically to interact with each other.
```

Comment:
I think it's clearer to associate the qualifiers exclusively with the *type* rather than the executable element it's attached to.

Reply:
* Qualifier on method: qualifies return type
* Qualifier on type: qualifiers the type the class operates on, usually.
  * On `ArgumentFactory`, qualifies input argument type
  * On `ColumnMapper<T>`, qualifies `T`
  * On `ColumnMapperFactory`, qualifies the mapped type
* Qualifier on parameter: qualifies parameter type